charts/postgres-operator/values-crd.yaml

```diff
@@ -93,10 +93,9 @@ configKubernetes:
   # template for database user secrets generated by the operator
   secret_name_template: "{username}.{cluster}.credentials.{tprkind}.{tprgroup}"
   # group ID with write-access to volumes (required to run Spilo as non-root process)
-  # spilo_fsgroup: 103
-
+  spilo_fsgroup: "103"
   # whether the Spilo container should run in privileged mode
-  spilo_privileged: false
+  spilo_privileged: "false"
```

Comment:
this change seem unnecessary / just cluttering the PR.

Reply:
yes please ignore. I am building on top of https://github.com/zalando/postgres-operator/pull/704 where we had similar conversations. I will rebase this PR when the other gets merged.